pkg/config/libpodConfig.go

```diff
@@ -262,7 +262,7 @@ func systemLibpodConfigs() ([]string, error) {
 			}
 			// TODO: Raise to Warnf, when Podman is updated to
 			// remove libpod.conf by default
-			logrus.Debugf("Found deprecated file %s, please remove. Use %s to override defaults.\n", path, containersConfPath)
+			logrus.Debugf("Found deprecated file %s, please remove. Use %s to override defaults.\n", Path(), containersConfPath)
```

Comment:
It seems the function `Path` is only used for logging but not the code path actually loading the configs, which might lead to a different path being logged and loaded.

Reply:
Well yes, but the code that does the actual search is also going to use /usr/share/containers/containers.conf for reading the distro configuration.  But we don't want to indicate to users that they should use that configuration.

Reply:
Sounds reasonable.